recipe/meta.yaml

```diff
@@ -49,8 +50,9 @@ requirements:
   run:
     - flann
     - qhull
-    - vtk * qt*
```

Comment:
i beleive that this would override the run_export, adding a constraint gives you "two different constraints". good enough for now.